.github/workflows/docker_release.yml

```diff
@@ -47,6 +47,18 @@ jobs:
           targets: base
           push: true
 
+      - name: Test base images
+        run: |
+          EXPECTED_VERSION=$(ct VERSION.txta)
```

Comment:
I am not sure if this is typo, but shouldn't this be `cat VERSION.txt` ?

Reply:
Oh yes! That's indeed wrong, fixing it. Nice catch.